package.json

```diff
@@ -40,18 +40,13 @@
   },
   "dependencies": {
     "@carbon/addons-website": "^0.0.1-alpha.23",
-    "@carbon/icons-react": "10.4.0-alpha.5390",
-    "@carbon/colors": "v10.1.1",
-    "@carbon/grid": "v10.1.1",
-    "@carbon/import-once": "v10.1.0",
-    "@carbon/layout": "v10.1.1",
-    "@carbon/type": "v10.1.1",
+    "@carbon/elements": "10.4.0",
```

Comment:
We might need to keep `@carbon/icons-react` as a dependency since it's not included in `@carbon/elements` by default.

Reply:
Oh, didn't realize. I'll update!